### DISCUSSION TITLE
Have you published a paper using GCAM?

### DISCUSSION THREAD

**Discussion by kvcalvin** [JGCRI/gcam-core#164] [Show and tell]
We'd like to update this page (https://jgcri.github.io/gcam-doc/references.html) to include papers from the GCAM community. If you have published a paper using GCAM, please post it in this discussion so we can get it added.

**Comment by crvernon** [comment]
Ryan McManamay (Baylor University), Henriette Jager (ORNL) and I just published some independent research in a paper titled "Global Biodiversity Implications of Alternative Electrification Strategies Under the Shared Socioeconomic Pathways" which utilized GCAM outcomes published in Graham et al. 2020.

> Ryan A. McManamay, Chris R. Vernon, Henriette I. Jager. Global Biodiversity Implications of Alternative Electrification Strategies Under the Shared Socioeconomic Pathways. Biological Conservation, 2021; 109234 DOI: 10.1016/j.biocon.2021.109234

See features:

- https://www.eurekalert.org/news-releases/644472
- https://www.sciencedaily.com/releases/2021/07/210713184401.htm
- https://www.labmanager.com/news/study-evaluates-biodiversity-impacts-of-alternative-energy-strategies-26287
- https://www.baylor.edu/mediacommunications/news.php?action=story&story=224058

**Comment by FeralFlows** [comment]
@kvcalvin Just to make sure I understand what is useful to post here, are you thinking that it's useful to post papers that may have been externally funded/led efforts by the broader GCAM community (i.e., by teams outside of PNNL/JGCRI)? That's how I interpreted your message after seeing the response from @crvernon. For example, we know all the GCIMS and IHESD papers, presumably, through internal documentation (e.g., https://jgcri.github.io/gcam-doc/references.html, though this does not seem comprehensive). But we might want to list papers here that are led and/or funded externally that might not be on the team's radar? Following that logic, here are a few recent papers:

> Wild, T.B., Khan, Z., Zhao, M., Suriano, M., Bereslawski, J., Roberts, P., Cassado, J., Gavino Novillo, M., Clarke, L., Hejazi, M., Miralles-Wilhelm, F., Munoz-Castillo, R., Vernon, C., Snyder, A., Yarlagadda, B., Birnbaum, A., Lamontagne, J., White, D., Ojeda-Matos, G. (2021). The Implications of Global Change for the Co-Evolution of Argentina’s Integrated Energy-Water-Land Systems. Earth’s Future. DOI: 10.1029/2020EF001970.

> Wild, T.B., Khan, Z., Clarke, L., Hejazi, M., Bereslawski, J., Suriano, M., Roberts, P., Casado, J., Miralles-Wilhelm, F., Gavino Novillo, M., Munoz-Castillo, R., Moreda, F., Yarlagadda, B., Lamontagne, J., Birnbaum, A. (2021). Integrated Energy-Water-Land Nexus Planning in the Colorado River Basin (Argentina). Regional Environmental Change. DOI: 10.1007/s10113-021-01775-1.

> Da Silva, S.R.S., Hejazi, M., Iyer, G., Wild, T.B., Binsted, M., Miralles-Wilhelm, F., Patel, P., Snyder, A., Vernon, C. (2021). Power Sector Investment Implications of Climate Impacts on Renewable Resources in Latin America and the Caribbean. Nature Communications.

> Delgado, R., Wild, T.B., Arguello, R., Clarke, L. (2020). Options for Colombia’s Mid-Century Deep Decarbonization Strategy. Energy Strategy Reviews. https://doi.org/10.1016/j.esr.2020.100525.

> Bataille, C., Vogt-Schilb, A., Jaramillo, M., Waisman, H., Briand, Y., Svensson, J., Lallana, F., Bravo, G., Le Treut, G., Lefevre, J., Delgado, R., Arguello, R., Wild, T.B., Clarke, L., Godinez, G., Quiros-Tortos, J., Howells, M., Soria, R., Carvajal, P., Schaeffer, R., Tovilla, J., Buira, D., Farbes, J., Jones, R., Gomez, X., Requejo, F., Collado, M., Ugarte, D. (2020). Net-zero deep decarbonization pathways in Latin America: challenges and opportunities. Energy Strategy Reviews. https://doi.org/10.1016/j.esr.2020.100510.

> Khan, Z., Wild, T.B., Carrazzone, M., Gaudioso, R., Mascari, M.P., Bianchi, F., Weinstein, F., Perez, F., Perez, W., Miralles-Wilhelm, F., Clarke, L., Hejazi, M., Vernon, C., Kyle, P., Edmonds, J., Munoz-Castillo, R., (2020). Integrated energy-water-land nexus planning to guide national policy: An example from Uruguay. Environmental Research Letters. https://doi.org/10.1088/1748-9326/ab9389.

**Reply by kvcalvin** [reply]
Yes! I'm trying to build up a complete list of GCAM papers, regardless of who did the work or funded them. Thanks for sharing!

**Reply by FeralFlows** [reply]
@kvcalvin Do we only want to collect papers that are using GCAM itself (since this conversation is on the gcam-core repository), or also papers that are part of the broader suite of interconnected tools? For example, there might be a paper that focuses on Xanthos or Demeter. It would be nice to maintain a list of such papers as well, even if it's a separate list, so that everyone can see how widely these various models are being used by the research community. Maybe @crvernon already does this for other tools.

**Reply by crvernon** [reply]
@FeralFlows I don't publicly report ancillary GCIMS model citations elsewhere, but we can certainty can start to do so!

**Reply by FeralFlows** [reply]
@crvernon Could we list publication on the repositories for the ancillary GCIMS models? For example, we could have a running list of papers that used xanthos on the xanthos repository? Obviously a ton of papers incidentally use Xanthos outputs if they're using GCAM, but we could just list papers (on the xanthos repository) where Xanthos played a key role, for example.

**Reply by crvernon** [reply]
@FeralFlows sure, we can do that.

**Reply by crvernon** [reply]
Hi @FeralFlows we now have a discussion board for Xanthos here:  https://github.com/JGCRI/xanthos/discussions

**Comment by fkanyako** [comment]
I just published our paper "Uncertainty analysis of the future cost of wind energy on climate change mitigation" using GCAM.
Kanyako, F., Baker, E. Uncertainty analysis of the future cost of wind energy on climate change mitigation. Climatic Change 166, 10 (2021). https://doi.org/10.1007/s10584-021-03105-0

**Comment by DLoughlin** [comment]
Most of our EPA GLIMPSE/GCAM team's publications have a PNNL/JGCRI author or co-author, so you likely have those on your list already. Here is one that doesn't: 

Babaee, S., Loughlin, D.H., and P.O. Kaplan (2020). Life cycle analysis of U.S. energy scenarios using a state-level integrated assessment model: GCAM-USA. Cleaner Engineering and Technology, https://doi.org/10.1016/j.clet.2020.100017.

Here are the manuscripts that have a JGCRI author or co-author: 

New:
Ou, Y., Iyer, G., McJeon, H., Cui, R., Zhao, A., O’Keefe, K.T.V., Zhao, M., Qiu, Y., and D.H. Loughlin (2023). State-level energy-water-land-health analysis of the US net-zero emissions goal. Energy and Climate Change, Oct. https://doi.org/10.1016/j.egycc.2023.100117.  

Loughlin, D.H., and C.G. Nolte (2023) Users’ Guide for GLIMPSE: a tool for integrated air-climate-energy planning. Office of Research and Development, U.S. Environmental Protection Agency, EPA 600/B-23/125. accessible via https://epa.gov/glimpse 

Prior years: 
Ou, Y., Kittner, N., Smith, S.J., Babaee, S., Nolte, C.G., and D.H. Loughlin (2021). Evaluating long-term emission impacts of large-scale electric vehicle deployment in the US using a human-earth systems model. Applied Energy, 300(2021), 117364. https://doi.org/10.1016/j.apenergy.2021.117364. 

Xia, J., Lu, X., Wang, S., Wang, T., Ding, D., Yu, S., Shindell, D., Ou, Y., Morawska, L., Li, S., Ren, L., Zhang, Y., Loughlin, D.H., Zheng, H., Zhao, B., Liu, S., Smith, K.R. and J. Hao (2020). The quest for improved air quality may push China to continue its CO2 reduction beyond the Paris Commitment. PNAS, Nov., DOI: https://doi.org/10.1073/pnas.2013297117.

Ou Y., West, J.J., Smith, S.J., Nolte, C.G., and D.H. Loughlin (2020). Air pollution control strategies directly limiting national health damages in the US. Nature Communications, (2020)11:957, DOI: https://doi.org/10.1038/s41467-020-14783-2.

Ou, Y., Smith, S.J., West, J.J., Nolte, C.G., and D.H. Loughlin (2019). State-level drivers of future fine particulate matter mortality for the United States. Environmental Research Letters,14(2019) 124071, DOI: https://doi.org/10.1088/1748-9326/ab59cb.

Ou, Y., Shi, W., Smith, S.J., Ledna, C.M., West, J.J., Nolte, C.G., and D.H. Loughlin (2018). Estimating environmental co-benefits of U.S. GHG reduction pathways using the GCAM-USA Integrated Assessment Model. Applied Energy, 216C(2018) pp. 482-493. DOI: https://doi.org/10.1016/j.apenergy.2018:02.122

Shi, W., Ou, Y., Smith, S.J., Ledna, C., Nolte, C.G., and D.H. Loughlin (2017). Projecting state-level air pollutant emissions using an integrated assessment model: GCAM-USA. Applied Energy, 208(2017), pp 511-521. DOI: https://doi.org/10.1016/j.apenergy.2017.09.122.
